packages/react/fast-components-react-base/src/stack-panel/stack-panel.props.ts

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import {
+    ManagedClasses,
+    StackPanelClassNameContract,
+} from "@microsoft/fast-components-class-name-contracts-base";
+import { Orientation } from "@microsoft/fast-web-utilities";
+
+export type StackPanelManagedClasses = ManagedClasses<StackPanelClassNameContract>;
+export type StackPanelUnhandledProps = React.HTMLAttributes<HTMLDivElement>;
+
+export interface StackPanelHandledProps extends StackPanelManagedClasses {
+    /**
+     * Whether items outside the viewport should be rendered or not
+     * default is true
+     */
+    virtualize?: boolean;
+
+    /**
+     *  the indexes of children that should always be rendered
+     */
+    neverVirtualizeIndexes?: number[];
+
+    /**
+     * The default span of each item in pixels (width in horizontal orientation, height in vertical)
+     * Provide a single value for panels where all items have the same value, or an array in the case where
+     * items have varying spans.
+     * default is 100
+     */
+    itemSpan?: number | number[];
+
+    /**
```

Comment:
The description vs the property name seem divergent to me - is this expecting a *pixel value* or a number of items? If the latter, perhaps using "count" instead of "length" would be more clear.

Reply:
Updated with "count"